src/test/java/com/ksm/wstbackoffice/endpoint/EndPoint.java

```diff
@@ -0,0 +1,11 @@
+package com.ksm.wstbackoffice.endpoint;
+
+public final class EndPoint {
+    public static final int PORT = 8082;
```

Comment:
you specify port here and in the application-test.yml. Why do you need it?

Reply:
I removed from application-test.yml in test directory because port equalled 0 in constructor 

public AddressControllerTest() {
        spec = new RequestSpecBuilder().
                setContentType(JSON).
                setBaseUri(RestAssured.DEFAULT_URI).
                setPort(EndPoint.PORT).
                addFilter(new ResponseLoggingFilter()).
                addFilter(new RequestLoggingFilter()).
                build();
    }

Reply:
You still have port here and in the application-test.yml. Why do you need it?

Reply:
I meant that I removed port from this file -> wst-backoffice/src/test/resources/application-test.yml
not from that file -> wst-backoffice/src/main/resources/application-test.yml here port needs described below